android/build.gradle

```diff
@@ -46,8 +55,8 @@ repositories {
 
 dependencies {
   implementation 'com.facebook.react:react-native:+'
-  implementation 'com.android.billingclient:billing:3.0.3'
-  implementation fileTree(dir: 'libs', include: ['*.jar'])
+  playImplementation 'com.android.billingclient:billing:3.0.3'
+  amazonImplementation fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
Splitting libraries per flavor. Reduces the bundle size and avoids bugs like the one we are seeing with OneSignal

Reply:
This crash is resolved with this PR: https://github.com/dooboolab/react-native-iap/issues/1291